### PR TITLE
fix: up to 12 events should be listed in cfs

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/4697
3. Call for Speakers
 - [x] Up to 12 events should be listed here and there should be a "more link" below to show more events on the cfs page if there are more events

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/controllers/index.js [hunk 1]
```diff
@@ -30,6 +30,11 @@ export default class IndexController extends Controller {
     return this.filteredEvents?.filter(event => !event.isPromoted);
   }
 
+  @computed('model.callForSpeakersEvents')
+  get cfsEvent() {
+    return this.model.callForSpeakersEvents.slice(0, 12);
```
Comment (iamareebjamal) [inline app/controllers/index.js:35]: page[size]=12 in CFS query

File: app/templates/index.hbs [hunk 2]
```diff
@@ -25,7 +25,7 @@
   {{#if this.model.callForSpeakersEvents}}
     <h2 class="main-heading">{{t 'Call for speakers'}}</h2>
     <div class="ui stackable three column grid">
-      {{#each this.cfsEvent as |event|}}
+      {{#each this.model.callForSpeakersEvents as |event|}}
```
Comment (iamareebjamal) [inline app/templates/index.hbs]: This shouldn't need a change now

Comment (maze-runnar) [reply app/templates/index.hbs]: `{{#each this.model.callForSpeakersEvents as |event|}}` or `{{#each this.cfsEvent as |event|}}`

Comment (iamareebjamal) [reply app/templates/index.hbs]: The change

Comment (maze-runnar) [reply app/templates/index.hbs]: I reverted it to previous didn't made any changes in this file.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**